recipes/xrandr.lua

```diff
@@ -131,7 +130,6 @@ local function xrandr()
                               function()
                                  state.timer:stop()
                                  state.timer = nil
-                                 state.iterator = nil
```

Comment:
Maybe, setting `state.index` to `nil` would make it always come again to the first `permutation` of screens. The way it is know, it always cycle through the permutations. Which behavior do you think it's better ?